docs/reference/synonyms/apis/delete-synonyms-set.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[delete-synoyms-set]]
+=== Delete Synonyms Set
+
+beta::[]
+
+++++
+<titleabbrev>Delete Synonyms Set</titleabbrev>
+++++
+
+Deletes a Synonyms Set.
+
+[[delete-synonyms-set-request]]
+==== {api-request-title}
+
+`DELETE _synonyms/<synonyms_set>`
+
+[[delete-synonyms-set-prereqs]]
+==== {api-prereq-title}
+
+* Requires the `manage_search_synonyms` cluster privilege.
+* You can only delete a Synonyms Set that is not in use by any <<analysis-synonym-graph-tokenfilter,synonym graph token filters>> and <<analysis-synonym-tokenfilter,synonym token filters>> that are included in an analyzer. See <<delete-synonym-set-analyzer-requirements>> for more information.
```

Comment:
I think we can say this shortly: 

You can only delete a Synonyms Set that is not in use by any index analyzer.

Reply:
Changed!